yt_dlp/YoutubeDL.py

```diff
@@ -2751,6 +2773,9 @@ def is_wellformed(f):
                 format['filesize_approx'] = int(info_dict['duration'] * format['tbr'] * (1024 / 8))
             format['http_headers'] = self._calc_headers(collections.ChainMap(format, info_dict))
 
+        # Safeguard against old/insecure infojson when using --load-info-json
+        info_dict['http_headers'] = self._remove_cookie_header(info_dict.get('http_headers')) or None
```

Comment:
as mentioned in https://github.com/yt-dlp/yt-dlp/pull/7546#discussion_r1257592266, not sure if this is necessary